test

BiteTheDDDDt opened the pull request:
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test <!-- At least one of them must be included. -->
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason <!-- Add your reason?  -->

- Behavior changed:
    - [ ] No.
    - [ ] Yes. <!-- Explain the behavior change -->

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. <!-- Add document PR link here. eg: https://github.com/apache/doris-website/pull/1214 -->

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label <!-- Add branch pick label that this PR should merge into -->

Commit:
support rf on set operator: be part

Thearas commented:
Thank you for your contribution to Apache Doris.
Don't know what should be done next? See [How to process your PR](https://cwiki.apache.org/confluence/display/DORIS/How+to+process+your+PR).

Please clearly describe your PR:
1. What problem was fixed (it's best to include specific error reporting information). How it was fixed.
2. Which behaviors were modified. What was the previous behavior, what is it now, why was it modified, and what possible impacts might there be.
3. What features were added. Why was this function added?
4. Which code was refactored and why was this part of the code refactored?
5. Which functions were optimized and what is the difference before and after the optimization?

BiteTheDDDDt commented:
run buildall

BiteTheDDDDt commented:
run buildall

englefly committed:
setop rf (fe part) (#49399)

### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test <!-- At least one of them must be included. -->
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
- [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason <!-- Add your reason?  -->

- Behavior changed:
    - [ ] No.
    - [ ] Yes. <!-- Explain the behavior change -->

- Does this need documentation?
    - [ ] No.
- [ ] Yes. <!-- Add document PR link here. eg:
https://github.com/apache/doris-website/pull/1214 -->

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label <!-- Add branch pick label that this PR
should merge into -->

BiteTheDDDDt commented:
run buildall

Commit:
fix

BiteTheDDDDt commented:
run buildall

Commit:
update fix

BiteTheDDDDt commented:
run buildall

englefly committed:
set legacy rf in RuntimeFilterContextV2

BiteTheDDDDt commented:
run buildall

englefly committed:
setop rf nullAware=true

englefly committed:
add-license

englefly commented:
run buildall

hello-stephen commented:
# BE UT Coverage Report
Increment line coverage `32.84% (22/67)` :tada:

[Increment coverage report](http://coverage.selectdb-in.cc/coverage/85d65170386d511dc9deaff08875e94ae783655f_85d65170386d511dc9deaff08875e94ae783655f/increment_report/index.html)
[Complete coverage report](http://coverage.selectdb-in.cc/coverage/85d65170386d511dc9deaff08875e94ae783655f_85d65170386d511dc9deaff08875e94ae783655f/report/index.html)
| Category          | Coverage           |
|-------------------|--------------------|
| Function Coverage | 50.89% (13621/26767) |
| Line Coverage     | 40.26% (118210/293615) |
| Region Coverage   | 38.94% (60064/154246) |
| Branch Coverage   | 33.85% (30211/89244) |

englefly committed:
update shape cases

englefly commented:
run buildall
